bindings/rust/aws-lc-sys-template/build.rs

```diff
@@ -0,0 +1,318 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+/* Copyright (c) 2022, Google Inc.
+ *
+ * Permission to use, copy, modify, and/or distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY
+ * SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION
+ * OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
+ * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE. */
+
+use std::fs::File;
+use std::io::{BufRead, BufReader, BufWriter, Write};
+use std::path::{Path, PathBuf};
+use std::process::Command;
+use std::{env, fs, io};
+
+use regex::Regex;
+
+fn modify_bindings(bindings_path: &PathBuf, prefix: &str) -> io::Result<()> {
+    // Needed until this issue is resolved: https://github.com/rust-lang/rust-bindgen/issues/1375
+
+    // This function modifies the generated bindings. The bindings are generated from our header files
+    // with symbol prefixing applied.
+    // This function will transform a line looking like this:
+    //
+    //     pub fn aws_lc_0_1_0_ERR_load_BIO_strings();
+    //
+    // Into lines like this:
+    //
+    //     #[link_name="aws_lc_0_1_0_ERR_load_BIO_strings"]
+    //     pub fn ERR_load_BIO_strings();
+    //
+    // This allows the function to appear with its original name (e.g., ERR_load_BIO_strings)
+    // in our bindings, while still being linked to the prefixed (e.g., aws_lc_0_1_0_ERR_load_BIO_strings)
+    // function name.
+
+    // The RE expression here has 3 capture groups.
```

Comment:
Is this [RAS (redundant acronym syndrome) syndrome](https://en.wikipedia.org/wiki/RAS_syndrome)? RE expression = Regular Expression expression?
```suggestion
    // The regular expression here has 3 capture groups.
```